libs/features/personalization/promo-utils.js

```diff
@@ -1,36 +1,70 @@
-import { getMetadata } from '../../utils/utils.js';
+import { getMetadata, getConfig } from '../../utils/utils.js';
 
 const GMTStringToLocalDate = (gmtString) => new Date(`${gmtString}+00:00`);
 
+const APAC = ['au', 'cn', 'hk_en', 'hk_zh', 'id_en', 'id_id', 'in', 'in_hi', 'kr', 'my_en', 'my_ms', 'nz', 'ph_en', 'ph_fil', 'sg', 'th_en', 'th_th', 'tw', 'vn_en', 'vn_vi'];
```

Comment:
Would it make sense to get these mappings from a json? I'm thinking about geo-expansion and the need to update the list of supported languages.

Reply:
i understand but what is the added value to separate it beside clarity for developers? Those lists are region mapping for promotion country, for now not used elsewhere.

Reply:
>and the need to update the list of supported languages.

for this use case its probably better to keep it in code similar to the list of locales in scripts.js.
hopefully geoexpansion PR will include changes to both lists at the same time, worst case we can always update it later.
json will not make it much easier